src/policies/interfaces/IRewardDistributor.sol

```diff
@@ -0,0 +1,145 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity >=0.8.20;
+
+import {IERC165} from "@openzeppelin-5.3.0/interfaces/IERC165.sol";
+
+/// @title  IRewardDistributor
+/// @notice Interface for the Reward Distributor contract
+interface IRewardDistributor is IERC165 {
+    // ========== EVENTS ========== //
+
+    /// @notice Emitted when a new Merkle root is set for an epoch
+    ///
+    /// @param  epochStartDate  The epoch start date for which the Merkle root is set
+    /// @param  merkleRoot      The Merkle root for that epoch's distribution
+    /// @param  rewardToken     The address of the reward token
+    event MerkleRootSet(uint256 indexed epochStartDate, bytes32 merkleRoot, address rewardToken);
+
+    /// @notice Emitted when a user successfully claims their rewards
+    ///
+    /// @param  user            The address of the user claiming rewards
+    /// @param  totalAmount     The total amount of rewards claimed
+    /// @param  rewardToken     The address of the reward token
+    /// @param  epochCount      The number of epochs claimed for
+    event RewardsClaimed(
```

Comment:
It feels like you could just combine these two events. If it's the vault token, pass it in as the reward token parameter. Or you could have the reward token and payout token separately.

Reply:
Thoughts?